bin/disco_vpc_ui.py

```diff
@@ -28,6 +28,9 @@ def parse_arguments():
                                help='What to call the new environment.')
     parser_create.add_argument('--type', dest='vpc_type', required=True,
                                help='What type of environment to create (as defined in config).')
+    parser_create.add_argument('--skip-enis', dest='skip_enis', action='store_const',
```

Comment:
By default, ENIs with static IP addresses would be automatically pre-allocated. This flag is to allow for stopping this behavior.

Another option is to reverse the behaviors between the default and the flag. But doing so would require changing the CI spin job to include the new flag.